troposphere/static/js/actions/ResourceActions.js

```diff
@@ -73,14 +76,32 @@ export default {
         );
 
         if (quota) {
+            var new_quota = (!quota.id) ? true : false;
             promises.push(
-                quota.save()
-                    .then(() => {
-                        Utils.dispatch( QuotaConstants.CREATE_QUOTA, { quota });
-                        return identity.set("quota", quota.toJSON())
-                                       .save({ quota: { id: quota.id } }, { patch: true });
+               // Skip quota updates (put/patch disabled!)
+               Promise.resolve((!quota.id) ? quota.save() : null).then(() => {
+                        debugger;
+                        if(new_quota) {
+                            Utils.dispatch( QuotaConstants.CREATE_QUOTA, { quota });
+                        }
+                        let approval_user = context.profile.get("username");
```

Comment:
This was really surprising to me. If you emulate as `new_user`. the app should consider you `new_user`, the exception to the rule is the few places where we want to know if emulation is occuring, in which case we could have `window.emulatee` i.e the staff account performing the emulation. If we move forward with `window.emulator` we must wrap every call to `context.profile.get("username")` with a check if emulating.